### PR TITLE
now scrolling to section after being pushed down by new data

### LINKED ISSUE #7069: Anchor links lead to wrong position on MozFest page
### Describe the bug
I'm using anchor links to send users to the different year heading on the [On-Demand page](https://www.mozillafestival.org/on-demand/previous-festivals) of the MozFest website. When a user clicks the link they are sent to the wrong position on the page.

### To Reproduce
1. Go to 'https://www.mozillafestival.org/on-demand/previous-festivals#mozfest-2019-healthy-ai'
2. You'll be brought to a point in the page before the MozFest 2019 | Healthy AI heading.

### Expected behavior
The link should position the page to the MozFest 2019 | Healthy AI heading

### Desktop (please complete the following information):
 - OS: MacOS 11.4
 - Browser: Chrome / Safari / FireFox

### Smartphone (please complete the following information):
 - Device: iPhone 12 Pro
 - OS: iOS 14.6
 - Browser: Safari

### Additional context
When watching the page load - it seems to load in the correct position, then the Pulse entries on the page load and push the headings down the page.

### PR DESCRIPTION
**Pull request by danielfmiranda**
Closes #7069

**Link to test page**: https://mozfest-foundation-s-pulse-list-0odhfg.mofostaging.net/en/tickets/#mozfest-2019-healthy-ai

**Steps to test**:

1. Please visit the test page above
2. Scroll down to find the different years of mozfest
3. Click on one of the years, it should take you to the same year section on the page.
4. It should render in the correctly location even after new data loads in, or, should scroll to the correct position after.

### CODE REVIEW

**Comment by danielfmiranda** [comment]
Hi @Pomax I am having a little bit of trouble getting the pulse entries to load and was wondering if you could help lead me in the right direction. I believe this is a cors issue because that is whats being returned in the log: 
image


However after looking at the env variables on heroku it looks like CORS is set to "*". I was wondering if you might know a way to resolve this so I can test with data? Thanks in advance!

File: source/js/components/pulse-project-list/pulse-project-list.jsx [hunk 1]
```diff
@@ -63,10 +63,24 @@ class PulseProjectList extends Component {
     projectXHR.send();
   }
 
+  // Giving users ability to link to pulse objects using an anchor link
+  // and having it render in the right place after recieving data.
+  scrollToLinkedPulseObject(){
+    const idOfLinkedPulseObject = (window.location.hash).substr(1)
+    const linkedPulseObject = document.getElementById(idOfLinkedPulseObject);
```
Comment (Pomax) [inline source/js/components/pulse-project-list/pulse-project-list.jsx:70]: Note that fragment ids are literally their own query selector, so we can just do this:
```suggestion
    const linkedPulseObject = document.querySelector(window.location.hash);
```

File: source/js/components/pulse-project-list/pulse-project-list.jsx [hunk 2]
```diff
@@ -63,10 +63,24 @@ class PulseProjectList extends Component {
...
+    linkedPulseObject.scrollIntoView()
```
Comment (Pomax) [inline source/js/components/pulse-project-list/pulse-project-list.jsx:71]: and then let's make sure to only scroll into view if there _was_ an element =)

```suggestion
    linkedPulseObject?.scrollIntoView()
```

**Review by Pomax — CHANGES_REQUESTED** [review]

**Review by Pomax — APPROVED** [review]
lgtm!

**Pull request merged**